### PR TITLE
Seznam článků

### PR DESCRIPTION
**Pull request by JiriNenahlo**

### CODE REVIEW

**Comment by JiriNenahlo** [comment]
Prepared common article list component for all required roles, should speed up progress a lot after it's finished and merged.

**Review by legas1 — CHANGES_REQUESTED** [review]
WIP, do not merge

File: src/App.tsx
Comment (legas1) [inline src/App.tsx]: remove the key plz

**Review by legas1 — CHANGES_REQUESTED** [review]

File: src/App.tsx
Comment (JiriNenahlo) [reply src/App.tsx]: fucking react

**Review by legas1 — APPROVED** [review]

**Pull request merged**
